docs/features/autoupdater.rst

```diff
@@ -3,6 +3,13 @@ Autoupdater
 
 Gluon contains an automatic update system which can be configured in the site configuration.
 
+Please mind, that asking users for consent is not implemented within the automatic update system.
+Also, users are neither asked nor notified when updates are available or installed.
+
+It appears reasonable to ask users for consent before administrators gain access to private devices and networks via automatic updates. This could be done before downloading, e.g. by providing different download options.
+Some community networks have guidelines regarding automatic update policies (e.g. Freifunk Memorandum of Understanding).
```

Comment:
The MoU only talks about consent, yet if you want to reference it please as quote & attribution.